configuration/appframework/patient_dashboard_extension.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "id": "ces.overallActions.downloadFrontPage",
+    "extensionPointId": "patientDashboard.overallActions",
+    "type": "link",
+    "label": "ces.downloadFrontPage",
+    "url": "module/pihcore/downloadReport.form?reportUuid=8be6802f-74a6-11f0-96a7-2ed2f846759c&patientId={{patient.patientId}}",
+    "icon": "fas fa-fw fa-download",
+    "order": 50
+  },
+  {
+    "id": "ces.overallActions.downloadClinicalHistory",
+    "extensionPointId": "patientDashboard.overallActions",
+    "type": "link",
+    "label": "ces.downloadClinicalHistory",
+    "url": "module/pihcore/downloadReport.form?reportUuid=004edb19-a3ac-11f0-9112-b688702e33e5&patientId={{patient.patientId}}",
+    "icon": "fas fa-fw fa-download",
+    "order": 51
+  }
```

Comment:
I had accidentally named this `system_administration_extension.json` when I added it initially, which isn't the right name, but more importantly would end up overwriting this file from config-pihemr:  https://github.com/PIH/openmrs-config-pihemr/blob/master/configuration/appframework/system_administration_extension.json , so any of those extensions were being accidentally removed.   You can likely confirm this on ces-ci @dmdesimone